PublicationsRouter/v2/sword-out/XWALK-EPrints.md

```diff
@@ -0,0 +1,139 @@
+# JPER Core Metadata to Dublin Core/RIOXX XML
+
+This document describes the crosswalk from the notification metadata fields received via the JPER API to the
+ XML format supplied to repositories via SWORDv2.
+ 
+The approach to the crosswalk is:
+
+* If there is a DC or DCTerms field to which the value can be written, do so
+* If there is also a RIOXX field which is different, do that one too
+
+This means that repositories which impelement one or other or both will have good changes of extracting useful
+metadata.
+
+For information about the schemas see:
+
+* [Dubin Core](http://dublincore.org/documents/dcmi-terms/)
+* [RIOXX](http://rioxx.net/v2-0-final/)
+
+The following table lists the notification fields in the JSON, then the DC/RIOXX fields it will be transformed to, and
```

Comment:
The following table lists the fields from the JPER API JSON notification, then the corresponding DC/RIOXX fields it will be transformed to, and any notes regarding the transformation: